sdk/core/Azure.Core/samples/Serialization.md

```diff
@@ -82,16 +82,27 @@ Given that explicit cast does not allow for serialization options we might also
 
 ## Using System.Text.Json
 
+In order to be integrate with the rest of the .NET ecosystem, Azure.Core supports System.Text.Json serialization. The following example demonstrates using System.Text.Json for serialization and deserialization
+If we go this route the IJsonSerializable interface will only be needed for compile time constraints and can most likely be methodless and renamed to IRehydratable.
```

Comment:
We should move this to a related issue

Reply:
I suspect there will be quite a few follow up issues once we decide (hopefully next week) on which direction we want to go.  If we go this route we will open this up to track it.